README.md

```diff
@@ -12,13 +10,108 @@ Simply run:
 
 $ inspec plugin install train-habitat
 
+## Using the plugin as a Train Transport
```

Comment:
Train Transport is used in this section, but not used elsewhere. Is this supposed to reference documentation in another repo?

Reply:
Alternatively, defining "Train Transport" may be helpful in understanding context.

Reply:
Train is a system for communicating with remote systems.  Each type of remote system requires a different Transport.  Some Transports ship with train, and some (like this one) ship as plugins.

So, this whole project is a train plugin; one uses it as a Train Trainsport to connect to habitat.

Reply:
So, to clarify, would the below work for amending line 3's description and provide that context to the rest of the document?
"train-habitat is a Train plugin and is used as a Train Transport to connect to Habitat installations."